leasing/serializers/rent.py

```diff
@@ -109,6 +109,12 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
+class IndexSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Index
+        fields = "__all__"
```

Comment:
I would declare which fields are used not just all.

Reply:
I just moved it up a bit in the file. But yes, I'll write out the fields.